src/gulpfile.js

```diff
@@ -107,7 +143,13 @@ gulp.task('copy:scripts', function() {
 });
 
 gulp.task('copy:vendor-css', function() {
-    return copyBowerFiles('**/*.css', [bootstrapSelectRoot + 'css/bootstrap-select.css'])
+    return copyBowerFiles('**/*.css',
+                          // FIXME: excluding minified CSS results in console error
+                          // about missing font awesome webfonts
+                          //'!**/*.min.css',
```

Comment:
Since we've apparently always been compiling CSS this way and it hasn't been an issue, I'm not too worried about it, but yes, let's make it an issue. I imagine with the redesign our font libraries and external CSS will change, so we might as well put off dealing with it until later.